src/components/Profile.tsx

```diff
@@ -0,0 +1,13 @@
+export function Profile() {
```

Comment:
when would this be displayed?

Reply:
tbh im not sure lol - i haven't quite figured out how we make it super smooth that u just have an "acct" you know -- it sort of will happen this way when i persist the form info and you fill out the "from" field, but i think it would be cool to see this stuff somewhere

roughly i was just imagining a lil icon in the top right (not a person icon, something more fun) that opens a portal to this stuff